core/state_transition.go

```diff
@@ -337,7 +337,14 @@ func (st *StateTransition) TransitionDb() (*ExecutionResult, error) {
 		effectiveTip = cmath.BigMin(st.gasTipCap, new(big.Int).Sub(st.gasFeeCap, st.evm.Context.BaseFee))
 	}
 	amount := new(big.Int).Mul(new(big.Int).SetUint64(st.gasUsed()), effectiveTip)
+	if london {
```

Comment:
How is this logic changing the Ethereum spec tests? Because this changes are not covered by those tests anymore so I am not sure if they should fail.

Reply:
We dont have test cases around it and we are currently not running any geth tests as well.

Reply:
This is a red flag and we need to address this ASAP.